test/functional/tests/fault_injection/test_standby_activate_different_cache_path.py

```diff
@@ -0,0 +1,88 @@
+#
+# Copyright(c) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+import pytest
+
+from api.cas import casadm
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_utils.size import Size, Unit
+from test_tools.dd import Dd
+from api.cas.cache_config import CacheStatus
+from api.cas.cache_config import CacheLineSize
+from api.cas.cli import standby_activate_cmd
+from api.cas.casadm_parser import get_caches
+
+cache_size = Size(16, Unit.GibiByte)
+block_size = Size(1, Unit.Blocks512)
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.nand, DiskType.optane]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+@pytest.mark.parametrizex("cache_line_size", CacheLineSize)
+def test_standby_activate_different_cache_path(cache_line_size):
+    """
+    title: test_standby_activate_different_cache_path
+    description: |
+        Symlink-activated CAS partitions distinction test.
+    pass_criteria:
+      - Cache is successfully activated with symlink on second partition.
+      - Cache is present and running.
+    """
+    with TestRun.step("Prepare two partitions with the same size."):
+        cache = TestRun.disks["cache"]
+        cache.create_partitions([Size(200, Unit.MebiByte), Size(200, Unit.MebiByte)])
+        active_cache_dev = cache.partitions[0]
+        standby_cache_dev = cache.partitions[1]
+        cache_id = 1
+
+    with TestRun.step("Start a regular cache instance on one of the partitions and stop it."):
+        cache = casadm.start_cache(active_cache_dev, force=True)
+        cache.stop()
+
+    with TestRun.step("Create a symbolic link to the second partition."):
+        standby_cache_dev_link = standby_cache_dev.get_device_link("/dev/disk/by-id")
```

Comment:
`get_device_link()` does not create new symbolic link, but looks for existing one, thus it's very likely to return the same path as `standby_cache_dev.path`. We need to create new distinct symbolic link here.